chapter-03/implementing-server-side-kotlin-development/src/test/kotlin/com/example/implementingserversidekotlindevelopment/domain/TitleTest.kt

```diff
@@ -1,8 +1,8 @@
 package com.example.implementingserversidekotlindevelopment.domain
 
-import arrow.core.Invalid
-import arrow.core.Valid
-import arrow.core.invalidNel
+import arrow.core.Either.Left
+import arrow.core.Either.Right
```

Comment:
[nits]

import は Either までに寄せる。

```suggestion
import arrow.core.Either
import arrow.core.Either
```